tests/raftstore/cluster.rs

```diff
@@ -458,6 +461,63 @@ impl<T: Simulator> Cluster<T> {
         assert!(status_resp.has_region_detail());
         status_resp.take_region_detail()
     }
+
+    pub fn partition(&mut self, s1: &HashSet<u64>, s2: &HashSet<u64>) {
+        for node_id in s1 {
+            let filter = new_partition_filter(s2);
+            self.sim.wl().push_trans_filter(*node_id, filter);
+        }
+        for node_id in s2 {
+            let filter = new_partition_filter(s1);
+            self.sim.wl().push_trans_filter(*node_id, filter);
+        }
+    }
+
+    pub fn heal_partition(&mut self, s1: &HashSet<u64>, s2: &HashSet<u64>) {
```

Comment:
If `heal_partition` is just to clear partition, I don't think we need to pass two hashset here. Just clear all filter is ok.